addon/styles/variables.scss

```diff
@@ -1,7 +1,31 @@
 // variables like color palette go here
-$light-gray: #a3a3a3;
-$lighter-gray: #f2f2f2;
+$light-gray: #c1c1c1;
+$lighter-gray: #f5f5f5;
 $gray: #5d5d5d;
-$dark-gray: #1a1a1a;
+$dark-gray: #222;
 $border-gray: #c3c3c3;
-$purple: #9472bc;
+$purple: #6a3ca3;
+$green: #58c949;
+$pink: #ff5c5c;
+
+
+// global properties
+// never change these directly: set them to local variables or set theme properties
+:root {
+  // font
+  --cs-input-font-family: var(--theme-font-family, 'Open Sans', Arial, Helvetica, sans-serif);
+  --cs-input-font-size: var(--theme-input-font-size, 16px);
+
+  // copy, background, borders, labels
+  --cs-input-background: var(--theme-input-background, #{$lighter-gray});
+  --cs-input-border: var(--theme-input-border, #{$border-gray});
+  --cs-input-border-radius: var(--theme-input-border-radius, 2px);
+  --cs-input-copy: var(--theme-input-copy, #{$dark-gray});
+  --cs-input-label: var(--theme-input-label, #{$gray});
+  --cs-input-label-focus: var(--theme-input-label-focus, #{$purple});
+  --cs-input-label-optional: var(--theme-input-label-optional, #{$gray});
+
+  // validation
+  --cs-input-valid: var(--theme-input-valid, #{$green});
+  --cs-input-invalid: var(--theme-input-invalid, #{$pink});
```

Comment:
What are your thoughts on using css variables vs sass variables? Are there best practices on when to use one vs the other?

Reply:
I did some research into this while we were discussing using Sass. Since Sass variables are precompiled, they're good to use for static values that won't need to change on the fly. CSS variables are good for properties that need to change dynamically in the browser, such as for user themes and responsive styles.